aasm.org

```diff
@@ -689,23 +689,59 @@ FF E0 ; jmp eax - E0 is 11 100 000 in binary
                         +- 11 is direct addressing
 FF 20 ; jmp [eax] - indirect jump
 
+EB cb Jump short, RIP = RIP + 8-bit displacement sign extended to 64-bits
+
+https://www.cyberforum.ru/asm-beginners/thread927866.html
+
+В случае косвенного перехода jmp в поле reg байта ModR/m
+хранится продолжение команды. Когда jmp ближний, там
+хранится 4, когда дальний - 5. Т.е. команда дальнего
+косвенного перехода выглядит так: 1111 1111 10 1---, где
```

Comment:
Maybe add more examples and clarify what parts of bit set refer to what parts of the command?